src/components/table/bl-table.stories.mdx

```diff
@@ -0,0 +1,168 @@
+import {html} from 'lit';
+import {Meta, Canvas, ArgsTable, Story} from '@storybook/addon-docs';
+import { UPDATE_STORY_ARGS } from "@storybook/core-events";
+
+<Meta
+		title="Components/Table/bl-table"
```

Comment:
Can we rename `bl-table` to `Table` here to make consistency across other components? Also, it'd be good to create pages for each component here for ease of access similar to [bl-select-option](https://6267ebb4dd54f5004a65f14a-elzlixfopr.chromatic.com/?path=/docs/components-select-selectoption--documentation).